pygdf/_gdf.py

```diff
@@ -544,20 +544,23 @@ def count_nonzero_mask(mask, size):
     return nnz[0]
 
 
+_GDF_COLORS = {
+    'green':    libgdf.GDF_GREEN,
+    'blue':     libgdf.GDF_BLUE,
+    'yellow':   libgdf.GDF_YELLOW,
+    'purple':   libgdf.GDF_PURPLE,
+    'cyan':     libgdf.GDF_CYAN,
+    'red':      libgdf.GDF_RED,
+    'white':    libgdf.GDF_WHITE,
+    'dark green': libgdf.GDF_DARK_GREEN,
```

Comment:
I'd propose `darkgreen` v/s `dark green` to be consistent with matplotlib's [named colors](https://matplotlib.org/2.0.2/examples/color/named_colors.html)

Reply:
Good to know! I'll change that.

Reply:
This is a super useful chart to have when I need to add more colors in the future.